5.x/crud-fields.md

```diff
@@ -466,6 +466,73 @@ Input preview:
 
 <hr>
 
+<a name="google-map"></a>
+### google_map
+
+Uses the Google Places API to generate a Map where user can navigate and select a position to store `Latitude` (**lat**), `Longitude` (**lng**) and the address string (**formatted_address**). By default it saves a string in dabase with `{lat: 123, lng: 456, formatted_address: 'Lisbon, Portugal'}`. 
```

Comment:
Sorry but no. This is NOT what the field does. Why are we writing from a technical perspective here? We should write from a USER perspective.

```suggestion
Shows a map and allows the user to navigate and select a position on that map (using the Google Places API). The field stores the latitude, longitude and the address string as a JSON in the database ( eg. `{lat: 123, lng: 456, formatted_address: 'Lisbon, Portugal'}`). If you want to save the info in separate db columns, continue reading below.
```